backend/maelstro/metadata/meta.py

```diff
@@ -165,12 +165,17 @@ def property_from_link(self, link_node: etree._Element, tag: str) -> str | None:
 
 
 class MetaZip(MetaXml):
-    def __init__(self, zipfile: bytes):
+    def __init__(self, zipfile: bytes, uuid=None):
```

Comment:
Maybe use typing for the argument if you feel like it
uuid: Optional[str] = None

(if I remember correctly - just getting things mixed up with typescript at the moment)

Reply:
I see, the CI even requires the typing, oops.

not sure if you have to import the `from typing import Optional` ...